### ISSUE TITLE
commands for PYLINK

### ISSUE THREAD

**Issue by darksis** [GLolol/PyLink#69]
everything look good but where the explain commands for Pylink?

**Comment by GLolol** [comment]
PyLink is not an average IRC bot. It's a PseudoService framework with relay capabilities similar to [Janus](http://ircwiki.mojeda.net/Janus), and therefore implementing any games or utility bot commands are the least of my concerns.

You will find some basic commands if you load the `commands` and `admin` plugins, which provide things like logging in and sending join/part/kick from **PyLink clients** only.

In short, there's nothing substantial when it comes to "commands", meaning this is really the wrong place if you're looking for that.

----

I hope this answers your question.

Seriously.
